app/jobs/cron/purge_old_email_event_job.rb

```diff
@@ -0,0 +1,7 @@
+class Cron::PurgeOldEmailEventJob < Cron::CronJob
+  self.schedule_expression = "every month at 3:00"
```

Comment:
```suggestion
  self.schedule_expression = "every week at 3:00"
```

pour faire un peu de rolling et éviter les fins de mois difficiles avec des dumps trop lourds